apidoc/Titanium/UI/Android/DrawerLayout.yml

```diff
@@ -223,3 +223,32 @@ examples:
         win.add(drawer);
         win.open();
         ```
+
+  - title: Alloy DrawerLayout example
+    example: |
+        The following code shows an Alloy version of the drawer-layout.
+        ``` xml
+        <Alloy>
+            <Window class='container'>
```

Comment:
Could you change this to `"container"` instead of single quotes? Besides that, looks good 👍

Reply:
sure, done